app/include/user_modules.h

```diff
@@ -59,6 +59,7 @@
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
 //#define LUA_USE_MODULES_TM1829
+#define LUA_USE_MODULES_TLS
```

Comment:
To me it would be more logical to have `#if defined(CLIENT_SSL_ENABLE) && defined(LUA_USE_MODULES_NET)` here rather than in `tls.c`. What do you think?